data/download_data.py

```diff
@@ -308,7 +308,7 @@ def download(resource_key: str, out_dir: str = None, keep_gzip: bool = False):
         resources = [k for k in RESOURCES_MAP.keys() if k.startswith(resource_key)]
         if resources:
             for key in resources:
-                download(key, out_dir)
+                download(key, out_dir, keep_gzip)
```

Comment:
Please see my comment in the previous commit. keep_gzip is not needed, just use an additional downloading configuration of the DB file with compressed attribute set to False